inc/namespace.php

```diff
@@ -1225,3 +1227,18 @@ function register_trace_qm_panel_menu_item( array $menu ) : array {
 
 	return $menu;
 }
+
+/**
+ * Set the PHP error message format to include the XRay trace ID.
+ *
+ * This requires the https://github.com/humanmade/error-message-format/ PHP
+ * extension to be installed. If it's not, this will have no effect.
+ *
+ * @return void
+ */
+function set_php_error_message_format() : void {
+	if ( function_exists( 'HM\\Platform\\XRay\\get_root_trace_id' ) ) {
+		$trace_id = XRay\get_root_trace_id();
+		ini_set( 'error_message_format', '{message} (Trace ID: ' . $trace_id . ')' );
```

Comment:
Can this return null/etc? (eg locally if there's no ID header)

Reply:
No, this always returns a string. If no trace ID is in the HTTP headers, it will generate one